types/sanitize-html/index.d.ts

```diff
@@ -8,9 +8,13 @@
 //                 A penguin <https://github.com/sirMerr>
 //                 Johan Davidsson <https://github.com/johandavidson>
 //                 Jianrong Yu <https://github.com/YuJianrong>
+//                 GP <https://github.com/paambaati>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
```

Comment:
I'm not convinced we should be locking this to TypeScript 2.2, e.g I know at least one place it's being used with TypeScript 3.x.

Reply:
@WillGibson This is the minimum version of TS that these typings are guaranteed to work with. It works fine on TS 3.x.

Reply:
@WillGibson as @paambaati mentioned is the minimum version. Here's a little [documentation](https://github.com/DefinitelyTyped/DefinitelyTyped/#i-want-to-use-features-from-typescript-21-or-above) about it.

Reply:
Thanks people.